sdk/core/core/inc/az_result.h

```diff
@@ -71,45 +71,48 @@ typedef enum
       _az_FACILITY_CORE,
       1), ///< Input argument does not comply with the requested range of values.
 
-  AZ_ERROR_BUFFER_OVERFLOW = _az_RESULT_MAKE_ERROR(
+  AZ_ERROR_INSUFFICIENT_SPAN_CAPACITY = _az_RESULT_MAKE_ERROR(
       _az_FACILITY_CORE,
-      2), ///< There is not enough space in the buffer provided.
-
-  AZ_ERROR_OUT_OF_MEMORY = _az_RESULT_MAKE_ERROR(
-      _az_FACILITY_CORE,
-      3), ///< Dynamic memory allocation request was not successful.
+      2), ///< There is not enough capacity in the span provided.
 
   AZ_ERROR_NOT_IMPLEMENTED
-  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 4), ///< Requested functionality is not implemented.
+  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 3), ///< Requested functionality is not implemented.
 
   AZ_ERROR_ITEM_NOT_FOUND
-  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 5), ///< Requestewd item was not found.
+  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 4), ///< Requested item was not found.
 
   AZ_ERROR_PARSER_UNEXPECTED_CHAR
-  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 6), ///< Input can't be successfully parsed.
+  = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 5), ///< Input can't be successfully parsed.
 
-  AZ_ERROR_EOF = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 7), ///< Unexpected end of the input data.
+  AZ_ERROR_EOF = _az_RESULT_MAKE_ERROR(_az_FACILITY_CORE, 6), ///< Unexpected end of the input data.
 
   // Platform
   AZ_ERROR_MUTEX = _az_RESULT_MAKE_ERROR(_az_FACILITY_PLATFORM, 1), ///< Mutex operation error.
 
+  AZ_ERROR_OUT_OF_MEMORY = _az_RESULT_MAKE_ERROR(
+      _az_FACILITY_PLATFORM,
+      2), ///< Dynamic memory allocation request was not successful.
```

Comment:
**Question:** What does this comment syntax mean? `///<` 

Is this something special, compared to, say:
```suggestion
      2), // Dynamic memory allocation request was not successful.
```

Reply:
That's Doxygen comment, simple "//" won't be included in the generated documentation.